src/components/ChatEntry.js

```diff
@@ -1,22 +1,39 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
 
 const ChatEntry = (props) => {
+  const likeFill = props.liked ? '❤️' : '🤍';
+  const onLike = () => {
+    props.updateLike(props.id);
+  }
+  const local = () =>{
+    if (props.sender === 'Estragon'){
+      return 'local'
+    } else {
+      return 'remote'
+    }
+  }
```

Comment:
Perfectly good way to determine the local user!

Reply:
Though I would pick a more descriptive name for this function. Perhaps something like `getLocalOrRemoteClass` even though that's a bit of a mouthful.